nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -106,26 +69,17 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_HOSTNAME = new PropertyDescriptor.Builder()
-            .name("SMTP_HOSTNAME")
-            .displayName("SMTP hostname")
-            .description("The hostname to be embedded into the banner displayed when an " +
-                    "SMTP client connects to the processor TCP port .")
-            .required(true)
-            .expressionLanguageSupported(false)
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .build();
-
-    protected static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
             .name("SMTP_MAXIMUM_CONNECTIONS")
             .displayName("Maximum number of SMTP connection")
             .description("The maximum number of simultaneous SMTP connections.")
             .required(true)
+            .defaultValue("1")
             .expressionLanguageSupported(false)
             .addValidator(StandardValidators.INTEGER_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_TIMEOUT = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_TIMEOUT = new PropertyDescriptor.Builder()
```

Comment:
I am not sure hat you're saying. If Subethasmtp is inadequate API, then we have to scrap it and use something different. But once again, let's keep in mind that the job of this processor is to simply open up a socket with attached application layer capable of speaking SMTP. That is all. For anything else use commercial server and attache to it via available mail protocols.